types/react-native/index.d.ts

```diff
@@ -9146,4 +9146,10 @@ declare global {
      * <code> if (__DEV__) console.log('Running in dev mode')</code>
      */
     const __DEV__: boolean;
+    /**
+     * This variable let's us declare things in global namespace
+     * Usage:
+     * <code>global.foo = 'bar';</code>
+     */
+    let global: any;
```

Comment:
I rarely define anything on global, so my recollection is a bit foggy, but this seems to be counter to what the above `global` typings try to achieve. Additionally, while you’ll be able to assign anything, you can’t afterwards use it anywhere and have the compiler provide you with insights/safety.

I _think_ you want a typings file in your project that adds to the global namespace those variables that your project needs. E.g. https://github.com/artsy/reaction/blob/master/src/Typings/global.d.ts

Reply:
I rarely define it too. I just wanted it to be defined by default, because that is how it works in javascript react-native right now. Also agree with adding things into global namespace thus closing this PR.

Reply:
Cool, thanks for taking the time, though!